DiagnosticReport/DiagnosticReport-GenomicReport-VariantAssessment.json

```diff
@@ -0,0 +1,111 @@
+{
```

Comment:
Could you add a presentedForm element, pointing to the wireframe report: https://cts5j3.axshare.com/?id=mhxwul&p=final_report_issued_test_complete&g=1

Reply:
Could you also create a DocumentReference example for PheobeSmitham (analagous to: https://simplifier.net/guide/FHIR-Genomics-Implementation-Guide/Home/Examples/DocumentReference?version=current) and add this to a mediaR5 extension to represent attaching genomic data files?